firestore/google/cloud/firestore_v1beta1/_helpers.py

```diff
@@ -837,6 +837,30 @@ def pbs_for_set(document_path, document_data, option):
     return write_pbs
 
 
+def _convert_simple_field_paths_with_leading_digits(field_paths):
+    """ Converts simple field path with integer beginnings to quoted field path
+
+    Args:
+        field_paths (List[str, ...]): A list of field paths
+
+    Returns:
+        new_field_paths (List[str, ...]):
+            The same list of field paths except field paths that start with
+            integers have been converted into quoted field paths. Simple field
```

Comment:
why *do* we convert to quoted field paths? Where is this specced out? Wouldn't this be surprising behavior?

Reply:
Good question.  I was thinking of just minimizing the changes needed to get to a working case.

@schmidt-sebastian
Is there an actual difference between a simple field path or a quoted field path or is it just "nomenclature"?  Should everything just be a quoted UTF-8 path since it's more inclusive anyway?  

Perhaps it's a memory issue for mobile?